docs/settings.rst

```diff
@@ -57,6 +57,80 @@ Example:
 
 .. _here: https://stripe.com/docs/api/python#create_plan
 
+DJSTRIPE_HIERARCHY_PLANS (={})
+===========================
+
+Payment plans levels. 
+
+Allow the user to set levels of access to the plans.
```

Comment:
I'm sorry @kavdev the truth is I have a really bad english jeje, I'm going to change that for:
Allow you to set levels of access to the plans.
As you suggest.